docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hollweg Dispersion Solver\n",
+    "\n",
+    "[tfds]: ../../api/plasmapy.dispersion.numerical.hollweg_.hollweg.rst\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "[hollweg1999]: https://doi.org/10.1029/1998JA900132\n",
+    "\n",
+    "This notebook details the functionality of the [hollweg()][tfds] function.\n",
+    "This function computes the wave frequencies for given wavenumbers and\n",
+    "plasma parameters based on the solution presented by [Bellan 2012][bellan2012]\n",
+    "to the [Hollweg 1999][hollweg1999] two fluid dispersion relation.\n",
+    "The dispersion relation assumes a uniform magnetic field, zero D.C\n",
+    "electric field, quasi-neutrality, wave propogation nearly perpendicular to\n",
+    "the magnetic field, a low-beta regime $c_{s} \\ll v_{A}$,\n",
+    "and low-frequency waves $\\omega \\ll \\omega_{ci}$ which equates to\n",
+    "\n",
+    "$$\n",
+    "    \\left( \\frac{\\omega^2}{{k_z}^2 {v_A}^2} - 1 \\right)\n",
+    "    \\left[\\omega^2\n",
+    "        \\left(\\omega^2 - k^2 {v_A}^2 \\right)\n",
+    "        - \\beta k^2 {v_A}^2 \\left( \\omega^2 - {k_z}^2 {v_A}^2 \\right)\n",
+    "    \\right]\n",
+    "    = \\omega^2 \\left(\\omega^2 - k^2 {v_A}^2 \\right) {k_x}^2 \\left(\n",
+    "    \\frac{{c_s}^2}{{\\omega_{ci}}^2} - \\frac{c^2}{{\\omega_{pe}}^2}\n",
+    "    \\frac{\\omega^2}{{k_z}^2 {v_A}^2} \\right)\n",
+    "$$\n",
+    "\n",
+    "where\n",
+    "\n",
+    "$$\\beta = c_{s}^2 / v_{A}^2$$\n",
+    "\n",
+    "$$k_{x} = k \\sin \\theta$$\n",
+    "\n",
+    "$$k_{z} = k \\cos \\theta$$\n",
+    "\n",
+    "$$\\mathbf{B_{o}} = B_{o} \\mathbf{\\hat{z}}$$\n",
+    "\n",
+    "$\\omega$ is the wave frequency, $k$ is the wavenumber\n",
+    ", $v_{A}$ is the Alfvén velocity, $c_{s}$ is the ion\n",
+    "sound speed, $\\omega_{ci}$ is the ion gyrofrequency, and\n",
+    "$\\omega_{pe}$ is the electron plasma frequency.\n",
+    "\n",
+    "The approach outlined in Section 3 of [Hollweg 1999][hollweg1999]\n",
+    "produces the above dispersion equation. The [hollweg()][tfds] function\n",
+    "numerically solves for the roots of this equation, namely the Fast,\n",
+    "Alfvén, and Acoustic wave modes,\n",
+    "\n",
+    "(Note: [Hollweg 1999][hollweg1999] asserts this expression is valid for\n",
+    "arbitrary $c_{s} / v_{A}$ and $k_{z} / k$. Contrarily,\n",
+    "[Bellan 2012][bellan2012] states in  Section 1.7 that due to the\n",
+    "inconsistent retention of the $\\omega / \\omega_{ci} \\ll 1$ terms\n",
+    "the expression can only be valid if both $c_{s} \\ll v_{A}$ and\n",
+    "the wave propgation is nearly perpendicular to the magnetic field.)\n",
```

Comment:
```suggestion
    "<div class=\"alert alert-info\">\n",
    "Note\n",
    "\n",
    "[Hollweg 1999][hollweg1999] asserts this expression is valid for arbitrary $c_{s} / v_{A}$ and $k_{z} / k$. Contrarily, [Bellan 2012][bellan2012] states in  Section 1.7 that due to the inconsistent retention of the $\\omega / \\omega_{ci} \\ll 1$ terms the expression can only be valid if both $c_{s} \\ll v_{A}$ and the wave propgation is nearly perpendicular to the magnetic field ($|\\theta - \\pi/2| \\ll 1$ radians).\n",
    "</div>\n",
    "\n",
    "The [hollweg()][tfds] function numerically solves for the roots of this equation, which correspond to the Fast, Alfvén, and Acoustic wave modes.\n",
```

The main part of this suggestion adds an admonition about Bellan's additional restrictions to this derivation.  I didn't change the wording of your original note, but I did add an inequality expression for theta.  In Jupyter Lab this renders like...

![image](https://user-images.githubusercontent.com/29869348/155032661-a0a574e0-b3a1-4d67-bdc2-cd9cc2345a61.png)

and on RTD it renders like...

![image](https://user-images.githubusercontent.com/29869348/155032711-84883517-c904-442f-9d47-2ef9212c7be4.png)

----

@namurphy You may like this (not well documented) bit about adding "admonitions" to a jupyter notebook.  [Currently `nbsphinx` only supports info (note) and warning "admonitions."](https://nbsphinx.readthedocs.io/en/latest/markdown-cells.html#Info/Warning-Boxes)